src/stepfunctions/steps/sagemaker.py

```diff
@@ -36,12 +36,12 @@ def __init__(self, state_id, estimator, job_name, data=None, hyperparameters=Non
             data: Information about the training data. Please refer to the ``fit()`` method of the associated estimator, as this can take any of the following forms:
 
                 * (str) - The S3 location where training data is saved.
-                * (dict[str, str] or dict[str, sagemaker.session.s3_input]) - If using multiple
+                * (dict[str, str] or dict[str, sagemaker.inputs.TrainingInput]) - If using multiple
```

Comment:
suggested edit: is it worth considering for the future is to specify the generic [`Dict`](https://docs.python.org/3/library/typing.html#typing.Dict) from typing (or even [`Mapping`](https://docs.python.org/3/library/typing.html#typing.Mapping))? I know these are just docstring comments, but in the future, you may want to include type hinting and the change will be minor in the future at that point:

```suggestion
                * (Dict[str, str] or Dict[str, sagemaker.inputs.TrainingInput]) - If using multiple
```

Reply:
I think an auto-formatter is a good idea.  I used `black` when I was ensuring I had applied the correct v2 SM upgrades, but the diff made it hard to compare changes so left it as is.

Those doc comments could be updated with a seperate PR as I suspect there are a few.